bitmovin/resources/models/codecconfigurations/h264_codec_configuration.py

```diff
@@ -6,7 +6,7 @@
 
 class H264CodecConfiguration(VideoCodecConfiguration, Serializable):
 
-    def __init__(self, name, bitrate, rate, profile, id_=None, description=None, custom_data=None, width=None,
+    def __init__(self, name, bitrate, profile, rate=None, id_=None, description=None, custom_data=None, width=None,
```

Comment:
Can we sample change the order here? What happens if someone calls this without providing the parameter names?